collections/copernicus-dem.yaml

```diff
@@ -95,22 +95,37 @@ CubeDimensions:
     extent:
       - -180
       - 180
+    reference_system: 4326
   y:
     type: spatial
     axis: y
     extent:
       - -85
       - 85
+    reference_system: 4326
   t:
     type: temporal
     extent:
       - '2010-12-12T05:04:33Z'
       - '2015-01-16T01:37:12Z'
-    step: P5D
   bands:
     type: bands
     values:
       - DEM
       - dataMask
+Summaries:
+  raster:bands:
+    - description: Heights in meters
```

Comment:
I'd prefer "Height (meters)" same as in the documentation. I think heights is not the best term here.

Reply:
I can change it to `Height (meters)`.

Just an explanation: I copied the descriptions from [**Available Bands and Data** part](https://docs.sentinel-hub.com/api/latest/data/dem/#available-bands-and-data) - the same part as I did for [S3 OLCI](https://github.com/eurodatacube/public-collections/pull/155) and [S3 SLSTR, S5P and MODIS](https://github.com/eurodatacube/public-collections/pull/159). The description that you mentioned is from [**Units**](https://docs.sentinel-hub.com/api/latest/data/dem/#units).

Reply:
Somehow missed that. In that case we can just leave it though I still find it sounds a bit strange.